pack/side/blbe_encounter.json

```
@@ -1,4 +1,422 @@
 [
+  {
+    "back_flavor": "Very little of Blackwater has been left untouched by the anomaly.",
+    "back_illustrator": "Emilio Rodriguez",
+    "back_name": "Quarantine Zone",
+    "clues": 2,
+    "code": "89001",
```

Suzi and her cards are 89001 to 89005, so please can this change? 

```suggestion
    "code": "89006",
```

---

I'll mark the changes so that the code matches the card number throughout.